presto-router/src/main/java/com/facebook/presto/router/cluster/ClusterStatusTracker.java

```diff
@@ -17,20 +17,17 @@
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.node.ObjectNode;
 import com.google.common.collect.ImmutableList;
-import com.google.common.collect.ImmutableSet;
 import com.google.inject.Inject;
+import org.weakref.jmx.Managed;
```

Comment:
This is removed in https://github.com/prestodb/presto/pull/24449, so it needs review & merge first